apps/hello-kubernetes/pom.xml

```diff
@@ -17,8 +17,8 @@
 		<java.version>11</java.version>
 		<spring-cloud.version>2020.0.2</spring-cloud.version>
 		<jkube.build.strategy>jib</jkube.build.strategy>
-		<jkube.generator.name>quay.io/lordofthejars/sb-hw:${project.version}</jkube.generator.name>
-		<jkube.enricher.jkube-controller.pullPolicy>Always</jkube.enricher.jkube-controller.pullPolicy>
+		<jkube.generator.name>quay.io/sunix/sb-hw:${project.version}</jkube.generator.name>
+		<jkube.createExternalUrls>true</jkube.createExternalUrls>
```

Comment:
with `openshift-maven-plugin` and openshift route this is not required ... but we have an issue at the moment with it https://github.com/eclipse/jkube/issues/1532